docs/eog/case-studies-cw20-timeline.rst

```diff
@@ -0,0 +1,145 @@
+      
+  - Followed up with sponsors and offered to renegotiate packages.
+  - Followed up with speakers and workshop facilitators, confirming any changes to their sessions, and provided office hours to test their setup and answer any questions.
+  - Contacted the event delivery team with new duties roster, held pre-event meetings to test the infrastructure and make sure everyone was comfortable with their role. 
+  - In the days before the event, we sent emails to registrants through Eventbrite reminding them to review the Code of Conduct, instructions to install and test the Zoom software, and information about how to prepare for certain sessions. 
+   
+    - We also stressed that CW20 would be as flexible on the day as possible, as many people were adjusting to a new way of working, caring responsibilities during the day, unstable internet, and to the associated emotional pressures. 
+    - We provided guidance for everyone to be kind, understanding and flexible. 
+      
+  - Links to the relevant Zoom room and note-taking document were sent via email through Eventbrite and Slack at the start of each day.
+   
+Resources
+  - We set up a uCONFLY instance for management of Discussion Group Reports, Mini-workship and Demo session notes, Collaborative Ideas and Hack Day ideas which also involved
+   
+    - Creating Google Doc templates for each of these document types.
+    - Generating a number of each document type for dissemination during CW20 so that participants would not be responsible for yet another platform.
+      
+  - Built the infrastructure needed to synchronise participants and run the unconference sessions such as
+   
+    - Collaborative notes documents with agenda and relevant links.
+    - Spreadsheets to disseminate the uCONFLY generated documents. 
+      
+  - Wrote documentation for the delivery team including
+   
+    - Zoom instructions 
+    - Duties roster
+    - Links index
+    - Start of day tasks for Event Lead/Chair
+    - Mini-workshop and demo session assignments
+      
+  - Made organiser slide deck.
+  
+Event run up actions
+  - Closed registration immediately upon decision to move event online.
+  - Issued full refunds to registrants who were no longer able to attend up to 7 days before the event start date.
+  - Issued partial refunds for the difference between the original priced and the new reduced price to registrants who still wanted to attend.
+  - Planned break-out groups distribution (sign up sheet for Discussion Groups session, randomly assigned for Collaborative Ideas session).
+  - Planned for video recording infrastructure (easy recording of sessions via Zoom).
+  - Had a Code of Conduct committee pre-event meeting.
+  - Constructed feedback form draft.
+
+
+Event Running Stage
+--------------------
+
+Month 0
```

Comment:
Might be worth saying it ran smoothly according to plan and any funnies that happened - I don't know if making everyone co-host to allow mingling for the hackday team formation is worth mentioning and that is an area we hope to improve in the future - i.e. mingling in online workshops.